linetools/analysis/voigt.py

```diff
@@ -0,0 +1,203 @@
+"""
+#;+ 
+#; NAME:
+#; voigt
+#;    Version 1.0
+#;
+#; PURPOSE:
+#;    Module for Voigt profiles
+#;   01-Nov-2014 by JXP
+#;      ## Heavily adapted from Ryan Cooke (e.g. alis)
+#;-
+#;------------------------------------------------------------------------------
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import sys
+import os
+import copy
+import warnings
+import pdb
+
+from astropy import units as u
+from astropy.units import Unit, Quantity
+from astropy import constants as const
+from astropy.modeling import FittableModel, Parameter
+
+from linetools.spectra.xspectrum1d import XSpectrum1D
+from linetools.spectralline import AbsLine
+
+#from xastropy.xutils import xdebug as xdb
+
+# The standard King model
+def voigtking(vin,a):
```

Comment:
This needs a reference, if possible.
It's from Ryan's Alis code, right?  (Is it a translation of the vpfit code?)